packages/workorder-angular/lib/workorder-form/workorder-form-controller.js

```diff
@@ -56,30 +58,42 @@ function WorkorderFormController($scope, $state, workorderApiService, workorderF
       } else {
         createUpdatePromise = workorderApiService.updateWorkorder(workorderToCreate);
       }
-
       createUpdatePromise.then(function() {
         //Finished with the update/create, go back to the list.
         workorderFlowService.workorderSelected(workorderToCreate);
       });
     }
   };
 
-  //TODO: Error handling
   $q.all([workorderPromise, workflowsPromise, workersPromise]).then(function(results) {
     self.model = results[0];
     self.workflows = results[1];
     self.workers = results[2];
 
 
-    if (self.model && self.model.startTimestamp) {
-      self.model.startDate = new Date(self.model.startTimestamp);
-      self.model.startTime = new Date(self.model.startTimestamp);
-      self.model.startTime.setMilliseconds(0);
-      self.model.finishDate = new Date(self.model.finishTimestamp);
-      self.model.finishTime = new Date(self.model.finishTimestamp);
-      self.model.finishTime.setMilliseconds(0);
+    if (self.model) {
+      if(self.model.assignee){
+        $http
+        .get("http://localhost:8001/api/users/" + self.model.assignee )
```

Comment:
@paolobueno - I left this only here for test purposes, I will need to move this to UserService, but this is currently being implemented by auth providers. I remember we had some code for WFM related user calls. 
It will be nice to move that implementation to core.